Tests/js/views/services/plugins/assets/ez-publishdraftplugin-tests.js

```diff
@@ -595,19 +634,122 @@ YUI.add('ez-publishdraftplugin-tests', function (Y) {
 
             this.view.fire('whatever:publishAction', {
                 formIsValid: true,
-                fields: fields
+                fields: fields,
+                serverSideErrorCallback: serverSideErrorCallback
```

Comment:
this seems to mean that `serverSideErrorCallback`, if that happens to be the case, that's a BC break